mapie/regression/time_series_regression.py

```diff
@@ -143,44 +141,94 @@ def fit(
         -------
         MapieTimeSeriesRegressor
             The model itself.
+
+        Raises
+        ------
+        ValueError
+            If the length of ``y`` is greater than
+            the length of the training set.
         """
-        self = super().fit(X=X, y=y, sample_weight=sample_weight)
-        self.conformity_scores_ = self._relative_conformity_scores(
+        check_is_fitted(self, self.fit_attributes)
```

Comment:
```suggestion
        warnings.warn(
                "WARNING: Deprecated method. "
                + "The method \"partial_fit\" is outdated. "
                + "Prefer to use \"update\" instead to keep "
                + "the same behavior in the future.",
                DeprecationWarning
            )
        check_is_fitted(self, self.fit_attributes)
```

Reply:
Thank you very much for the suggestions for improving the code!